src/models.py

```diff
@@ -0,0 +1,67 @@
+import datetime
+from sqlalchemy import Column, Integer, String, DateTime, ForeignKey, BigInteger, DECIMAL
+from sqlalchemy.orm import relationship, backref
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+# class Dummy(Base):
+#    __tablename__ = "dummy"
+#    id = Column(Integer, primary_key=True, index=True)
+#    dummy = Column(String(250),)
+
+
```

Comment:
Since you already implemented other classes, I think this can be removed, right?
```suggestion
```